pepper-apis/studybuilder-cli/studies/lms/i18n/es.json

```diff
@@ -41,6 +41,34 @@
       "required_hint": "Please choose a province"
     }
   },
+
+  "somatic_results": {
+    "sr_name": "What We’ve Learned from $ddp.isGovernedParticipant(\"Your Child's\",\"Your\") Tumor (somatic) DNA",
+    "sr_title": "What We’ve Learned from $ddp.isGovernedParticipant(\"Your Child's\",\"Your\") Tumor (somatic) DNA",
+    "sr_header": "About $ddp.isGovernedParticipant(\"Your Child's\",\"Your\") LMS",
+    "summary": {
+      "created": "View a printable document that provides information on what we’ve learned from the sequencing of $ddp.isGovernedParticipant(\"your child's\",\"your\") stored tumor samples.",
+      "in_progress": "View a printable document that provides information on what we’ve learned from the sequencing of $ddp.isGovernedParticipant(\"your child's\",\"your\") stored tumor samples.",
+      "complete": "View a printable document that provides information on what we’ve learned from the sequencing of $ddp.isGovernedParticipant(\"your child's\",\"your\") stored tumor samples.",
+    },
+    "introduction": {
+      "p1": """
+      Thank you for being a part of the Leiomyosarcoma Project. When you enrolled, you indicated that you would like to receive information from the sequencing of the DNA found in $ddp.isGovernedParticipant("your child's","your") tumor samples (also known as “somatic” DNA).
+      This information is now available if you would still like to see it.</br></br>
+      If for any reason you would like this information permanently removed from your dashboard, please contact the study team at  <a href="mailto:info@lmsproject.org" class="Link">info@lmsproject.org</a>,
+      or call us at <a href="tel:651-403-5556" class="Link">651-403-5556</a>.
+      """
+    },
+    "questions": {
+      "show_result": {
+        "prompt": "We have information about what we've learned from the sequencing of $ddp.isGovernedParticipant(\"your child's\",\"your\") tumor sample(s). Would you like to have access to this document? </br></br> <i>If you select No, you will not be able to download this document from your dashboard in the future without reaching out to the study team.</i>",
+      }
+    },
+    "info": "<p>What we've learned from sequencing $ddp.isGovernedParticipant(\"your child's\",\"your\") tumor DNA</p>",
+    "info_disagree": "<p>Thank you for your response. If you change your mind and would like to view/download this document in the future, please contact the study team at info@LMSproject.org or 651-403-5556."
```

Comment:
Should the email and phone numbers be links?

Reply:
should be. updated. thank you